tests/Colopl/Spanner/Console/WarmupCommandTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Copyright 2019 Colopl Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Colopl\Spanner\Tests\Console;
+
+use Colopl\Spanner\Connection;
+
+class WarmupCommandTest extends TestCase
+{
+    public function test_no_args(): void
+    {
+        foreach (['main', 'alternative'] as $name) {
+            $conn = $this->getConnection($name);
+            assert($conn instanceof Connection);
+            $this->setUpDatabaseOnce($conn);
```

Comment:
It seems that `TestCase::setUpDatabaseOnce()` [only handles one call](https://github.com/colopl/laravel-spanner/blob/master/tests/Colopl/Spanner/TestCase.php#L106-L107), but is the `alternative` connection setup handled in this case?

Reply:
this should be fixed now.

Reply:
Confirmed.
It seems that `$databasePrepared` is not necessary in the current code.

https://github.com/colopl/laravel-spanner/blob/3d1e37406b0dcbfc5ba53bf1190e6d09f6f211e7/tests/Colopl/Spanner/TestCase.php#L33

Reply:
Thanks, it has been removed.